frontend/@mf-types/@mf/modelRegistry/compiled-types/src/app/context/ModelRegistrySelectorContext.d.ts

```
@@ -0,0 +1,15 @@
+    modelRegistriesLoaded: boolean;
+    modelRegistriesLoadError?: Error;
+    modelRegistries: ModelRegistry[];
+    preferredModelRegistry: ModelRegistry | undefined;
+    updatePreferredModelRegistry: (modelRegistry: ModelRegistry | undefined) => void;
+};
+type ModelRegistrySelectorContextProviderProps = {
+    children: React.ReactNode;
+};
+export declare const ModelRegistrySelectorContext: React.Context<ModelRegistrySelectorContextType>;
+export declare const ModelRegistrySelectorContextProvider: React.FC<ModelRegistrySelectorContextProviderProps>;
+export {};
```

Biome (1.9.4)

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)